scoop_protocol/scoop_protocol.go

```diff
@@ -43,6 +43,11 @@ type LoadCheckResponse struct {
 	ManifestURL string
 }
 
+//Scoop healthcheck for ingester healthcheck json unmarshal
```

Comment:
We don't generally annotate for usage, as usage will probably change faster than Scoop protocol.